pkgs/misc/emulators/retroarch/default.nix

```diff
@@ -44,11 +51,11 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  meta = with stdenv.lib; {
+  meta = {
     homepage = http://libretro.org/;
     description = "Multi-platform emulator frontend for libretro cores";
     license = licenses.gpl3;
-    platforms = stdenv.lib.platforms.linux;
-    maintainers = with maintainers; [ MP2E edwtjo ];
+    platforms = with platforms; [ linux darwin ];
```

Comment:
Both `linux` and `darwin` are lists, so this should be `linux ++ darwin`, strictly speaking.